features/index_page.feature

```diff
@@ -0,0 +1,15 @@
+Feature: As a visitor
+So that I can access the site
+I need to be able to see the index page
+
+Background:
+  Given there is a menu called "Lunch"
```

Comment:
Yes, the test would not understand name in @menu.name in the menu.erb file. It seems there needs to be a menu for the page to be present. Adding the background made the code work.